handlers/organizations.go

```diff
@@ -44,6 +44,13 @@ func CreateOrEditOrganization(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(org.Name) == 0 || len(org.Name) > 20 {
+		fmt.Printf("invalid organization name %s\n", org.Name)
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode("Error: organization name must be present and should not exceed 20 character")
```

Comment:
@ecurrencyhodler  i have changed the error message from what was mentioned in the issue to make more sense. let me know if you want me to revert back to the once mentioned in the issue